evaluation/README.md

```diff
@@ -0,0 +1,17 @@
+# Kubernetes based Evaluation Pipeline
+This folder contains files needed to setup a kubernetes based evaluation
+pipeleline on GCP.
+
+## Source tree overview
+
+| File / Folder                    | Description                                                               |
+| :------------------------------- | :------------------------------------------------------------------------ |
+| **`cron`**                       | folder with files for cron job (docker image, k8s configs, ...)           |
```

Comment:
nit: you use periods in most of the lines (description column), but not in all.

Reply:
fixed ...

Reply:
here as well: `Base docker image for the evaluation (mainly to speed up compilation)   `